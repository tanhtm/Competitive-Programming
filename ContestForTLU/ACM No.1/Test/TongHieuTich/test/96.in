53
-911 -672
5 331
-137 969
953 40
-668 489
61 -959
427 -940
-993 -373
397 635
317 -590
-910 168
559 570
410 130
-88 -859
798 440
-813 108
94 -538
127 299
818 -612
-534 659
140 64
-922 -749
-188 -756
-863 -963
500 224
522 636
599 -969
627 -372
-788 543
-291 139
-894 -49
11 94
669 835
822 559
-527 651
-23 -682
-225 317
455 -706
754 231
324 -221
-407 171
81 119
769 904
-938 -84
-400 -526
583 -429
-659 -7
579 134
304 -563
-681 565
-821 -452
535 -247
124 -2
