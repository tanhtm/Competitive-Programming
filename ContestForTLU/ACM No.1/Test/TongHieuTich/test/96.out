94584
-27384
-230048
226652
51775
-228990
-175317
211730
-61404
-61902
199969
-3104
37800
-182534
110801
162326
-70152
-18318
73645
-37281
3876
72270
-134048
-45650
49956
-33003
-145040
63686
81523
16340
199208
-2178
-62416
90800
-36518
-116148
-12466
-72852
128788
14033
34102
-1900
-56463
218197
-29169
38962
108558
79321
-56138
36134
117434
56304
3843
